server/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM mcr.microsoft.com/dotnet/sdk:8.0 as builder
 WORKDIR /source
 COPY . .  
 RUN dotnet restore
-RUN dotnet publish -c Release -o /out GameControl.Server.csproj
+RUN dotnet publish -c Debug -o /out GameControl.Server.csproj
```

Comment:
Is changing this to debug in checked in code intentional?

Reply:
Yeah since this is the local development Dockerfile - I was planning on making this an environment variable the next time we change this